HackYourFuture-CPH/fp-class12
# Pushing Header feature

# Description

In this PR i have created one component for title on the header which is working perfectly fine with knobs and i have one another component for modal which is functional but i did not added knobs for that.

Fixes # 15
You are welcome to check my code and If you find something please let me know 🙂 

# How to test?
You can test it in storybook UI by running : npm run storybook
You can see it in action how heading changes when page is changed by running dev mode : npm run dev

# Checklist

- [x] I have merged the latest version of the source branch (typically develop) into this branch before pushing my changes
